python/lsst/ts/electrometer/controller.py

```diff
@@ -244,7 +245,7 @@ async def start_scan(self):
         await self.send_command(f"{self.commands.set_buffer_size(50000)}")
         await self.send_command(f"{self.commands.select_device_timer()}")
         await self.send_command(f"{self.commands.next_read()}")
-        self.manual_start_time = time.time()
+        self.manual_start_time = time.monotonic()
```

Comment:
I suggest using lsst.ts.utils.current_tai for all timestamps.

It's perhaps overkill for measuring durations, but values like this often end up being logged or published in SAL topics (even if you don't do that now).

Reply:
Done by me since Eric is on vacation.